### PR TITLE
Replace methods for configuration param

### PR DESCRIPTION
**Pull request by Themanwhosmellslikesugar**

### CODE REVIEW

File: bin/server.py [hunk 1]
```diff
@@ -74,6 +74,8 @@ def __init__(self, application, request, **kwargs):
         self._base_packages_query = {}
         self._selected_packages = []
 
+        self._configuration = dict(defaults.CONFIGURATION)
```
Comment (karpa4o4) [inline bin/server.py:77]: * Add `time_zone` item to `defaults.CONFIGURATION`;
* Rename `hostname` to `host_name`, `SSID` to `WPA_SSID` and `PSK` to `WPA_PSK`;

File: bin/server.py [hunk 2]
```diff
@@ -145,13 +147,21 @@ async def change_root_password(self, request, password):
 
     @only_if_initialized
     @remote
-    async def sync_configuration(self, request, image_configuration_params):
-        request.ret(READY)
+    async def get_configuration_main_params(self, request):
```
Comment (karpa4o4) [inline bin/server.py]: Implement only two remote procedures: `get_configuration` and `set_configuration`. `set_configuration` procedure should change only those parameters that are already in `self._configuration` dict.

File: bin/server.py [hunk 3]
```diff
@@ -145,13 +147,17 @@ async def change_root_password(self, request, password):
...
-    async def sync_configuration(self, request, image_configuration_params):
-        request.ret(READY)
+    async def get_configuration_params(self, request):
+        request.ret(self._configuration)
 
     @only_if_initialized
     @remote
-    async def get_default_configuration(self, request):
-        request.ret(defaults.CONFIGURATION)
+    async def set_configuration_params(self, request, configuration):
```
Comment (karpa4o4) [inline bin/server.py]: The class attribute is named `_configuration`, not "configuration_params". Therefore, it is logical to call the procedures `get_configuration`/`set_configuration` (what i suggested in my first comment) and not to arrange a zoo.

File: blackmagic/db.py [hunk 4]
```diff
@@ -31,9 +32,22 @@ def __init__(self, user_id, name, device_name, distro_name, flavour):
...
         self.image_id = str(uuid.uuid4())
 
+    def _get_configuration_props(self):
+        props = {}
+        for prop_key, prop_value in self._configuration.items():
+            if (not self._configuration['enable_wireless']
+                    and prop_key in defaults.WIRELESS_CONFIGURATION_KEYS):
+                continue
+
+            prop_name = f'PIEMAN_{prop_key.upper()}'.upper()
```
Comment (eugulixes) [inline blackmagic/db.py]: Why do we upper twice?

File: blackmagic/db.py [hunk 5]
```diff
@@ -72,11 +78,13 @@ def dump_sync(self):
         image.flavour = 'C'
         image.status = self._status
 
-        configuration_props = self._get_configuration_props()
-        image.props = {
+        props = {
             'PIEMAN_INCLUDES': ','.join(self._selected_packages),
-            **configuration_props,
         }
+        props.update(self._get_configuration_props())
```
Comment (eugulixes) [inline blackmagic/db.py]: I guess the changes are redundant. You simply needed to replace `image.props` for `props` and that's it.

**Pull request merged**